tuyishimekyrie/control-system-frontend
# Add Continuous Integration (CI) Pipeline for Code Linting

## Summary

This PR introduces a CI pipeline specifically for code linting, ensuring that all code changes adhere to our coding standards before being merged.

## Changes

Configured the CI pipeline to run linting checks automatically on each pull request.
Integrated the linting step into the existing CI workflow to enforce consistent code quality.
Updated configuration files as necessary to align with our linting rules.

## How to Test

Open a new pull request and observe that the CI pipeline runs linting checks automatically.
Ensure that the pipeline fails if code does not pass the linting standards, and passes when it does.

## Notes

This addition will help catch potential coding issues early in the development process and maintain a clean codebase.
No other functionality has been modified, only CI configuration.
